src/main/java/com/amazonaws/services/neptune/io/KinesisConfig.java

```diff
@@ -20,12 +20,13 @@ public class KinesisConfig {
 
     private final Stream stream;
 
-    public KinesisConfig(String streamName, String region, LargeStreamRecordHandlingStrategy largeStreamRecordHandlingStrategy) {
+    public KinesisConfig(String streamName, String region, LargeStreamRecordHandlingStrategy largeStreamRecordHandlingStrategy, boolean enableAggregation) {
         this.stream = (StringUtils.isNotEmpty(region) && StringUtils.isNotEmpty(streamName)) ?
                 new Stream(
                         new KinesisProducer(new KinesisProducerConfiguration()
+                                .setAggregationEnabled(enableAggregation)
                                 .setRegion(region)
-                                .setRateLimit(100)
+                                .setRateLimit(120)
```

Comment:
Is there any potential negative impact to increasing the rate limit? Is this something that should be user configurable?

Reply:
120 is still more conservative than the default, which is 150. This value encourages a constant feed of records down to the stream. At this point, I'd rather not expose a lot of producer configuration unless there's a strong need for it from users.

Reply:
Thanks that sounds good